src/main/webapp/help-snapshotId.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    The ID or name of the snapshot from which to execute the operation. If the snapshot_id is not, selects the latest
+    snapshot
```

Comment:
The ID or name of the Snapshot from which to execute the step. If the snapshot_id is not specified, then DCT will automatically select the latest Snapshot.